tools/oprt2/pkg/config/config.go

```diff
@@ -21,18 +21,62 @@ type Authenticator struct {
 	// Not implemented
 }
 
-// PackageManager defines package manager configuration.
-type PackageManager struct {
+// FileManager defines a storage backend for storing and pulling files.
+type FileManager struct {
 	// Not implemented
 }
 
-// Attune defines Attune publishing configuration.
-type Attune struct {
+// GPGProvider defines a source of GPG keys.
+type GPGProvider struct {
+	// Not implemented
+}
+
+// AttuneAPTPackagePublisher publishes packages for an APT repo via Attune.
+// If this is used, then the global Attune configuration must also be provided.
+type AttuneAPTPackagePublisher struct {
 	// Authentication defines Attune authentication configuration.
 	Authentication Authenticator
-	// ParallelUploadLimit is the maximum number of packages to try to upload to the Attune
-	// control plane at once. If unset, there will be no upload limit.
-	ParallelUploadLimit uint
+	// GPG is how GPG keys will be retrieved for repo signing. Optional.
+	GPG *GPGProvider
+}
+
+// DiscardAPTPackagePublisher doesn't actually publish any APT packages.
+// Useful for dry-runs.
+type DiscardAPTPackagePublisher struct{}
```

Comment:
If this is for a no-op publisher used in tests/dry-run, why does it even need a config?